docs/Project.toml

```diff
@@ -1,14 +1,15 @@
 [deps]
 Cairo = "159f3aea-2a34-519c-b102-8c37f9878175"
-Colors = "5ae59095-9a9b-59fe-a467-6f913c188581"
+CategoricalArrays = "324d7699-5711-5eae-9e2f-1d82baa6b597"
```

Comment:
Unclear why you added this.  We don't like to have a lot of extra dependencies in Gadfly.

Reply:
Ignore my comment, since CategoricalArrays is already a dependency (I don't use it much).